src/ContainerManagement.js

```diff
@@ -0,0 +1,79 @@
+import Dependency from "./Dependency"
+
+const rall = /.+/
+
+export default {
```

Comment:
Do we no longer need to do:

```javascript
this.dependencies = {}
```

in order to do 

```javascript
this.dependencies[name] = "hello world";
```

Shouldn't that `Cannot set property __ of undefined`?